config/compression.go

```diff
@@ -0,0 +1,81 @@
+package config
+
+import "fmt"
+
+type Compression struct {
+	Request  ReqCompressionConfig  `mapstructure:"request"`
+	Response RespCompressionConfig `mapstructure:"response"`
+}
+
+// CompressionConfig defines if compression is enabled and what type of compression to use
+type ReqCompressionConfig struct {
```

Comment:
IMHO: Since this code is in the `config` package, there is no need to include "Config" in the name. Same comment for "RespCompressionConfig".

Reply:
Makes sense. Will remove the `Config` suffix